apps/mobile/src/components/register-adoption-form/adoption-form.hooks.test.tsx

```diff
@@ -46,3 +49,107 @@ describe('useMultiStepNavigation', () => {
   });
 });
+
+jest.mock('../../shared/hooks/use-locale')
+jest.mock('native-base')
+jest.mock('react-hook-form')
+
+describe('useFormValidation', () => {
+  beforeEach(jest.clearAllMocks)
+
+  describe('when validateField is called', () => {
+    describe('when field is valid', () => {
+      it('returns validateField as true', async () => {
+        jest.mocked(useLocale).mockReturnValue({ t: jest.fn() })
+        jest.mocked(useToast).mockReturnValue({ ...useToast(), show: jest.fn(), isActive: jest.fn(), })
+        jest.mocked(useFormContext).mockReturnValue({
+          ...useFormContext(),
+          trigger: () => Promise.resolve(true),
+          formState: {
+            errors: { name: { type: 'string.empty' } } as Record<string, any>
+          } as any
+        })
+
+        const { result } = renderHook(() => useFormValidation());
+
+        await act(async () => {
+          const isValid = await result.current.validateField('name')
+          expect(isValid).toBeTruthy();
+        })
+      });
+    })
+
+    describe('when field is invalid', () => {
+      it('returns validateField as false', async () => {
+        jest.mocked(useLocale).mockReturnValue({ t: jest.fn() })
+        jest.mocked(useToast).mockReturnValue({ ...useToast(), show: jest.fn(), isActive: jest.fn(), })
+        jest.mocked(useFormContext).mockReturnValue({
+          ...useFormContext(),
+          trigger: () => Promise.resolve(false),
+          formState: {
+            errors: { name: { type: 'string.empty' } } as Record<string, any>
+          } as any
+        })
+
+        const { result } = renderHook(() => useFormValidation());
+
+        await act(async () => {
+          const isValid = await result.current.validateField('name')
+
+          expect(isValid).toBeFalsy();
+        })
+      });
+
+      describe('whe isActive returns false', () => {
+        it('calls show toast', async () => {
+          const show = jest.fn()
+
+          jest.mocked(useLocale).mockReturnValue({ t: (key) => key })
+          jest.mocked(useToast).mockReturnValue({ ...useToast(), show, isActive: () => false })
+          jest.mocked(useFormContext).mockReturnValue({
+            ...useFormContext(),
+            trigger: () => Promise.resolve(false),
+            formState: {
+              errors: { name: { type: 'string.empty' } } as Record<string, any>
+            } as any
+          })
+
+          const { result } = renderHook(() => useFormValidation());
+
+          await act(async () => {
+            await result.current.validateField('name')
+          })
+
+          expect(show).toBeCalledWith({
+            description: 'REGISTER_ADOPTION.FORM.NAME_STRING_EMPTY',
+            id: "adoption-form-toast",
+          });
+        });
+      })
+
+      describe('whe isActive returns true', () => {
+        it('does calls show toast', async () => {
+          const show = jest.fn()
+
+          jest.mocked(useLocale).mockReturnValue({ t: (key) => key })
```

Comment:
Please define a default value for this mock at the top, this way you don't need to repeat the intended value across the tests.